pkg/operator/ceph/object/rgw-probe.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+PROBE_TYPE="{{ .ProbeType }}"
+PROBE_PORT="{{ .Port }}"
+PROBE_PROTOCOL="{{ .Protocol }}"
+
+# standard bash codes start at 126 and progress upward. pick error codes from 125 downward for
+# script as to allow curl to output new error codes and still return a distinctive number.
+USAGE_ERR_CODE=125
+PROBE_ERR_CODE=124
+# curl errors: 1-123
+
+STARTUP_TYPE='startup'
+READINESS_TYPE='readiness'
+case "$PROBE_TYPE" in
```

Comment:
Since this is embedded (no user input), could we avoid the input checks? Then no need for this case statement.

Reply:
That's probably a good idea to keep it simpler.